### ISSUE TITLE
net/http warning is not suppressed 

### ISSUE THREAD

**Issue by samnissen** [jeremyevans/ruby-warning#2]
```ruby
# spec_helper.rb
Warning.ignore(/net\/http: warning\: Content\-Type did not set\; using application\/x-www-form-urlencoded/)
```

```ruby
$ rspec spec/some_file.rb
....net/http: warning: Content-Type did not set; using application/x-www-form-urlencoded
...
```

#### Meta
warning 0.10.1
ruby 2.4.1

**Comment by jeremyevans** [comment]
In ruby 2.4, `Kernel#warn` does not call `Warning.warn`.  This bug was fixed in ruby 2.5.  Please upgrade to ruby 2.5.  Alternatively, you can override `Kernel#warn` to call `Warning.warn` in ruby 2.4.

**Comment by samnissen** [comment]
👍
